mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -697,7 +691,14 @@
     <string name="cookie_banner_exception_panel_description_state_on_for_site">%1$s will clear this site’s cookies and refresh the page. Clearing all cookies may sign you out or empty shopping carts.</string>
 
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is off for a site, this is shown as part of the cookie banner panel in the toolbar. The placeholder will be updated with the app name. -->
-    <string name="cookie_banner_exception_panel_description_state_off_for_site">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %1$s may accept all cookies to dismiss the banner.</string>
+    <string name="cookie_banner_exception_panel_description_state_off_for_site">%1$s can try to automatically reject cookie requests.</string>
```

Comment:
This string was changed without a new ID. Which means now we have about 45 errors
https://github.com/mozilla-l10n/mozl10n-linter/actions/runs/3907552132/jobs/6676885089

This change should be reverted, and a new string ID used.

Reply:
@iorgamgabriel : could you please help with updating this? I merged the string but it's creating errors now